privacyidea/lib/utils/__init__.py

```diff
@@ -61,7 +61,7 @@
 # character lists for the identifiers in the pin content policy
 CHARLIST_CONTENTPOLICY = {"c": string.ascii_letters, # characters
                           "n": string.digits,        # numbers
-                          "s": string.punctuation}   # special
```

Comment:
Can we leave the ``string.punctuation`` and somehow fix the tests/functionality?

Reply:
I tried to, but I can give it another shot.

Reply:
see #2238